tests/deployment/lib/archetype_definitions/archetype_definition_customer_landing_zone.tmpl.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We should probably remove this one and just specify the `default_empty` archetype definition ID where this is currently used.

Reply:
Updated in commit [Cleanup archetype definitions and variables](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/pull/22/commits/cb736560540e4718a92e62ba256290f3382aa89f)